atom-u2i-hackathon/lib/u2i-hackathon.coffee

```diff
@@ -183,7 +183,9 @@ module.exports =
           "Error while checking your solution:\n" + error.message)
 
   onOutputs: (outputs) ->
-    requestSuccessful = @challengesService.checkSolution outputs
```

Comment:
We're not passing subtractedPoints to backend anywhere so we're always sending default value which is 0. I believe it's a bug or sending it does not make sense at all

Reply:
There was a plan to write a plugin that subtracts points e.g. for each keystroke but it looks like we haven't implemented it.